kaggle_environments/utils.py

```diff
@@ -15,6 +15,7 @@
 import json
```

Comment:
This file is all style and formatting updates, reviewers can likely skip this file.